content/docs/next-release/troubleshooting.md

```diff
@@ -3,4 +3,6 @@ title: Troubleshooting
 description: Solve commonly encountered issues
 weight: 10
 draft: true
```

Comment:
Remove draft?

Reply:
I removed, but doesn't seem to have an effect?

Reply:
how do you mean? It shows up under 1.9 now

![image](https://user-images.githubusercontent.com/3523016/51553819-d19dd480-1e41-11e9-9721-26297817d47f.png)